src/Compilers/Core/Portable/InternalUtilities/Hash.cs

```diff
@@ -171,6 +171,7 @@ internal static int GetFNVHashCode(byte[] data)
             return hashCode;
         }
 
+#if !CODE_STYLE
```

Comment:
sorry, i feel dumb.  

> This is the only compiler-layer change. ReadOnlySpan<T> isn't available in the code style layer.

Where is readonlyspan used in this code?

Reply:
➡️ It appears when you merge this branch into master. 😄

Reply:
ah gotcha.  thanks!